test/performance/perf-kinesis/mzcompose.yml

```diff
@@ -32,9 +32,30 @@ services:
     init: true
     command: -w 4
 
-  grafana:
+  dashboard:
     mzbuild: dashboard
     environment:
       - 'MATERIALIZED_URL=materialized:6875'
     ports:
       - *grafana
+
+mzconduct:
+  workflows:
+    load-test:
```

Comment:
do we test perf-kinesis in CI? Is it worth adding a ci workflow similar to chbench to make it easier to debug locally?

Reply:
We don't currently test perf-kinesis in CI, but I imagine we will?